src/include/target/DIY_2400_TX_ESP32_SX1280_E28.h

```diff
@@ -28,15 +30,29 @@
 #define GPIO_PIN_OLED_SDA       32
 #define GPIO_PIN_OLED_SCK       33
 #define GPIO_PIN_OLED_RST       U8X8_PIN_NONE
+#elif defined(HAS_TFT_SCREEN)
+#define  GPIO_PIN_TFT_MOSI      32
+#define  GPIO_PIN_TFT_SCLK      33
+#define  GPIO_PIN_TFT_RST       16
+#define  GPIO_PIN_TFT_DC        22
+#define  GPIO_PIN_TFT_CS        2
+#define  GPIO_PIN_TFT_BL        25
 #endif
 #define GPIO_PIN_RCSIGNAL_RX    13
 #define GPIO_PIN_RCSIGNAL_TX    13
 #define GPIO_PIN_LED_WS2812     15
 #define GPIO_PIN_FAN_EN         17
 
+#ifdef HAS_FIVE_WAY_BUTTON
+#define GPIO_PIN_FIVE_WAY_INPUT1     39
+#define GPIO_PIN_FIVE_WAY_INPUT2     35
+#define GPIO_PIN_FIVE_WAY_INPUT3     34
+#endif
```

Comment:
This is not the recommended way to wire up a 5 way joystick, the better option is to use a voltage divider on a single ADC pin.